docker-compose.yml

```diff
@@ -21,38 +21,59 @@ services:
     stdin_open: true
     tty: true
     volumes:
-      - app_data:/app/data/dev
-      - log:/app/log
-      - tmp:/app/tmp
```

Comment:
The reason we have volumes is either to share files between docker containers ([named volume](https://docs.docker.com/storage/volumes/)) or to share files between the host and the docker containers ([bind mount](https://docs.docker.com/storage/bind-mounts/)).

For a named volume like `repos` for example, this is because it needs to be shared between the `rails` and `ssh` containers. For a named volume like `packs`, this needs to be shared between `rails` and `webpacker`. etc.

`tmp` was a named container but does not need to be shared between any containers. However, we do want to be able to see any changes to the `tmp/` folder in the container reflected on the host filesystem. This is handled by the `.:/app:cached` line which sets a bind mount for the root of the project (which includes `tmp/` as subdirectory). Similarly, the `scanned_exam` subfolder does not need to be a volume for the same reasons as `tmp/`.

By including `tmp/` (or `scanned_exams/`) as a separate named volume all we are doing is creating a volume which does not buy us any functionality, creates a new object that docker has to handle and maintain, making the structure more complex than it needs to be. 

I am happy to move these changes to a separate PR but I see these changes not as a change in functionality but as cleanup; the same way I would remove unnecessary code/tests as part of a PR that is not specifically a "clean up" PR but touches the file that requires cleaning.  This PR involves the creation of several new, necessary volumes so it makes sense to clean up the old, unnecessary volumes at the same time.

Reply:
Thanks for the update. Please move these changes into a separate PR. Doing so will make it easier for me to review the changes, including "cleanup".

Reply:
added